cpp/ps0003/Task3.cpp

```diff
@@ -4,18 +4,20 @@ using namespace std;
 
 int main()
 {
+    // prompting user for number 2/2
     float x;
 
     cout << "Enter your number: \n";
     cin >> x;
 
+    // checking if the number is odd or even 3/3
     if (x % 2 == 0)
     {
-    cout << "This number is even \n";
+        cout << "This number is even \n";
     }
     else
     {
-    cout << "This number is odd \n";
+        cout << "This number is odd \n";
     }

```

Comment:
- [x] used modulo operation by 2 `%2` to check if the number is even or odd
- [x] number is even when modulo result returns `0` and odd when returns `1`
- [x] used `std::cout` in a right way to print result in console